msgraph/endpoints.go

```diff
@@ -0,0 +1,47 @@
+package msgraph
+
+import (
+	"golang.org/x/oauth2"
+)
+
+var tenantLoginEndpoint string
+var tenantMSGraphEndpoint string
```

Comment:
It doesn't look like these 2 are used outside of their respective functions below. Could we move them into being locally scoped for each function?